### ISSUE TITLE
Camel case issue on new filter properties of 2.6.1

### ISSUE THREAD

**Issue by jonathanhuet** [neo4j-graphql/neo4j-graphql-js#257]
Hi,

**Goal**
Get an entity list with filtering via a 1to1 relation added in 2.6.1

**Expected result**
Get all the entities that are related to an other entity

**Current result**
No nodes returned

**How to reproduce**

_Schema_

A Survey can have multiple SurveyBatch
```
type Survey {
  id: ID
  name: String!
  surveyBatches: SurveyBatch @relation(name: "BATCHES", direction: "OUT")
}

type SurveyBatch {
  id: ID
  survey: Survey @relation(name: "BATCHES", direction: "IN") 
}

```

_GraphQL query_

```
{
  SurveyBatch(filter: {survey: {id: "a5be5836-9476-4dd9-941e-926c46ff4b6d"}}) {
    id
  }
}
```
_Generated Cypher Query_
```
MATCH (`surveyBatch`:`SurveyBatch`) WHERE (EXISTS((`surveyBatch`)<-[:BATCHES]-(:Survey)) AND ALL(`survey` IN [(`surveybatch`)<-[:BATCHES]-(`_survey`:Survey) | `_survey`] WHERE (`survey`.id = $filter.survey.id))) RETURN `surveyBatch` { .id } AS `surveyBatch`

```

**Reason** 
The `surveybatch` in the `IN`clause is not well camelCased it should be `surveyBatch`

Note: IMHO, I think this issue has a big impact
PD: So many thanks for the version 2.6.1, filtering is a huuuuge improvement

**Comment by jonathanhuet** [comment]
I managed to fix it by modifying `translate.js` function `buildRelatedTypeListComprehension`at line [2382](https://github.com/neo4j-graphql/neo4j-graphql-js/blob/master/src/translate.js#L2382) from
```
return `[(${thisTypeVariable})${relationDirection === 'IN' ? '<' : ''}-[${
```

to

```
return `[(${variableName})${relationDirection === 'IN' ? '<' : ''}-[${
```

I hope that this can help :)

**Comment by johnymontana** [comment]
Thanks @jonathanhuet! Would you like to submit that in a pull request? :-)

**Comment by johnymontana** [comment]
Fixed by #325 Please reopen if you're still seeing issues.
